maap/maap.py

```diff
@@ -256,13 +256,76 @@ def cancelJob(self, jobid):
         job.id = jobid
         return job.cancel_job()
 
-    def listJobs(self, username=None, page_size=None, offset=None):
-        if username==None and self.profile is not None and 'username' in self.profile.account_info().keys():
+    def listJobs(self, username=None, 
+                       algo_id=None, 
+                       end_time=None, 
+                       get_job_details=True, 
+                       offset=None, 
+                       page_size=None, 
+                       priority=None, 
+                       queue=None,
+                       start_time=None,
+                       status=None,
+                       tag=None, 
+                       version=None):
+        """
+        Returns a list of jobs for a given user that matches query params provided.
+
+        Args:
+            username (str): Platform user.
+            algo_id (str): Algorithm type.
```

Comment:
Are these optional?

Reply:
Yes. Turns out username is _technically_ optional.